examples/io.ipynb

```diff
@@ -70,7 +70,15 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "me.io.entities_to_csv(\"example.csv\", Ms=Ms, T=T, angle=theta_angle, demag_factor=demag_factor, comment=comments)"
+    "me.io.entities_to_csv(\n",
+    "    \"example.csv\",\n",
+    "    description=\"Example csv file.\\nThe description can have multiple lines.\\n\\nLines can also be empty.\",\n",
```

Comment:
- the argument `description` does no longer exist
- you have to pass it as positional argument and should no longer be allowed to use a keyword argument